test/torch/mpc/test_fss.py

```diff
@@ -2,7 +2,9 @@
 import torch as th
 import numpy as np
 
-from syft.frameworks.torch.mpc.fss import DPF, DIF, n
+from syft.frameworks.torch.mpc.fss import DPF, DIF, N
+
+# NOTE: the FSS class is also tested in the Sycret package
```

Comment:
One test with pre-provided material would be helpful as well!
Suggestion (approx):
```python
sy.local_worker.crypto_store.force_preprocessing = True

sy.local_worker.crypto_store.provide_primitives(
    workers=(data_owner, model_owner),
    n_instances=8,
    op="fss_comp"
)

encryption_kwargs = dict(
    workers=(data_owner, model_owner), # the workers holding shares of the secret-shared encrypted data
    crypto_provider=crypto_provider, # a third party providing some cryptography primitives
    protocol="fss", # the name of the crypto protocol, fss stands for "Function Secret Sharing"
)

t = th.ones(8)
x = t.encrypt(**encryption_kwargs)
z = (x > 0)

sy.local_worker.crypto_store.force_preprocessing = False
```

Reply:
Good idea! I tried something here: https://github.com/OpenMined/PySyft/blob/ae0c927f5d94a1920081d4ee377de91fd28322a8/test/torch/mpc/test_fss.py#L143, but I'm not familiar with the high level encrypt()/decrypt() interface so I'm not sure it makes sense.